### PR TITLE
chore: add the tutorial of lauching a HTAP cluster based on PostgreSQL, MyDuck server and PGPool

### LINKED ISSUE #182: Work around for statements querying "pg_catalog"
Some clients connected based on PostgreSQL protocol will query the views created under "pg_catalog", which are used for fetching the status of PostgreSQL. Unfortunately, MyDuck can not handle those queries. So we add a temporarily solution to circumvent these queries by intercepting the queries and return the mock data.

However, not all views are mocked now, we just mock those queries sent by the client we have tested so far.

### LINKED ISSUE #183: add mock parameter status message packages

### PR DESCRIPTION
**Pull request by VWagen1989**
This PR aims to add the tutorial of lauching HTAP for PostgreSQL protocol using PostgreSQL instance, MyDuck server and a proxy named [PGPool-II](https://www.pgpool.net/docs/pgpool-II-4.2.5/en/html/index.html).

To launch the HTAP cluster successfully, this PR also solve some issues about the compatibilities to the PGPool. But these are work around solutions. Maybe we should use a better approach to solve these problems.
* resolve #182 
* resolve #183

### CODE REVIEW

File: catalog/internal_tables.go [hunk 1]
```diff
@@ -74,11 +74,51 @@ func (it *InternalTable) SelectStmt() string {
...
+	for _, c := range it.KeyColumns[1:] {
+		b.WriteString(", ")
+		b.WriteString(c)
+	}
+	b.WriteString(", ")
+	b.WriteString(it.ValueColumns[0])
+	for _, c := range it.ValueColumns[1:] {
+		b.WriteString(", ")
+		b.WriteString(c)
+	}
```
Comment (fanyang01) [inline catalog/internal_tables.go:91]: `SELECT *` is enough.

File: pgserver/connection_handler.go [hunk 2]
```diff
@@ -430,6 +482,107 @@ func (h *ConnectionHandler) handleQuery(message *pgproto3.Query) (endOfMessages
...
+		}
+		return "", err
+	}
+
+	return setting, nil
+}
+
+// TODO(sean): This is a temporary work around for clients that query the views from schema 'pg_catalog'.
+// Remove this once we add the views for 'pg_catalog'.
+func (h *ConnectionHandler) handledPGcatalogQueries(statement string) (bool, error) {
```
Comment (fanyang01) [inline pgserver/connection_handler.go:545]: -> handlePgCatalogQueries

**Review by fanyang01 — COMMENTED** [review]
Thanks! Looks exciting.

**Review by TianyuZhang1214 — APPROVED** [review]
It seems we have some conflicts. But it's fun, I'll solve them in my branch. LGTM!

File: pgserver/connection_handler.go (hunk 2 above)
Comment (VWagen1989) [reply pgserver/connection_handler.go]: fixed

**Pull request merged**